src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -650,10 +650,12 @@ export function getEventContent(event) {
         description: '',
       };
     default:
-      return {
-        title: 'Unknown event',
-        description: '',
-      };
+      Raven.captureMessage('appeals-unknown-event', {
+        extra: {
+          eventType: event.type
```

Comment:
Would any other information about the event be helpful to track?

Reply:
:thinking: I don't think so, but @cmgiven might have an idea. What do you think, Chris?

Reply:
I think that would do it. This helpfully flags a mismatch between the API and the front-end. Knowing is most of the battle, and we can quickly figure out the rest once we know there's an issue.